TAS_Engine/Convert/Environment_oM/InternalGain.cs

```diff
@@ -52,45 +54,123 @@ public static List<BHE.Gain> ToBHoM(this TBD.InternalGain tbdInternalGain)
             tasData.Add("InternalGainDescription", tbdInternalGain.description);
             tasData.Add("InternalDomesticHotWater", tbdInternalGain.domesticHotWater);
 
+            //Lighting
+            BHE.Gain lightGain = new BHE.Gain();
+            lightGain.Name = "L " + tbdInternalGain.name;
+            lightGain.GainType = BHE.GainType.Lighting;
+
+            BHP.GainPropertiesLighting lightingGain = new BHP.GainPropertiesLighting();
+            lightingGain.GainType = BHE.GainType.Lighting;
+            lightingGain.GainUnit = BHE.GainUnit.WattsPerSquareMetre;
+            lightingGain.RadiantFraction = tbdInternalGain.lightingRadProp;
+            lightingGain.ViewCoefficient = tbdInternalGain.lightingViewCoefficient;
+            tasData.Add("targetIlluminance", tbdInternalGain.targetIlluminance);
```

Comment:
I would prefer to see all `tasData` additions in one place rather than spread out - can you move this up to line 56 with the other `tasData` items.

Reply:
done